src/components/DataViewFilterMultiselect/DataViewFilterMultiselect.tsx

```diff
@@ -115,7 +115,6 @@ function DataViewFilterMultiselect(props: DataViewFilterMultiselectProps) {
 				onExited={onExited}
 			>
 				<DataViewFilterMultiselectDropdownContent
-					value={value}

```

Comment:
Just curious, was this prop not being used by the `DataViewFilterMultiselectDropdownContent`?